sdk/include/psdk/ntddndis.h

```diff
@@ -316,7 +316,19 @@ typedef ULONG NDIS_OID, *PNDIS_OID;
 #define OID_802_11_PRIVACY_FILTER               0x0D010119
 #define OID_802_11_BSSID_LIST_SCAN              0x0D01011A
 #define OID_802_11_WEP_STATUS                   0x0D01011B
+/* New name is supposed to better reflect the extended set of encryption status */
+#define OID_802_11 ENCRYPTION_STATUS            OID_802_11_WEP_STATUS
```

Comment:
There shouldnt be a space here, but an underscore? (between `11` and `ENCRYPTION`)

Reply:
Oh, good catch. Committing now